configs/ote/custom-counting-instance-seg/resnet50_maskrcnn/template.yaml

```diff
@@ -0,0 +1,53 @@
+# Description.
+model_template_id: Custom_Instance_Segmentation_MaskRCNN-ResNet50
```

Comment:
I would give some counting problem -related name.

Reply:
updated but not sure if it's clear enough